app/js/components/LabOrdersList/LabOrdersList.jsx

```diff
@@ -7,13 +7,131 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react';
+import { connect } from 'react-redux';
+import cn from 'classnames';
+import moment from 'moment';
+import Loading from '../Loading';
+import SortableTable from '../SortableTable';
+import { fetchLabOrders } from '../../actions/labOrderAction';
+import "react-table/react-table.css";
+import "./index.scss";
+
+
+
+const Cell = ({columnName, value}) => {
+  if (columnName === "EMR ID") {
+    const emrID = value.patient.display.split('-')[0];
+    return (
+      <div className= "table-cell emr-id">
+        <span>{emrID}</span>
+      </div>
+    );
+  }
+  if (columnName === "NAME") {
+    const displayName = value.patient.display.split('-')[1];
+    return (
+      <div className= "table-cell order-date">
+        <span>{displayName}</span>
+      </div>
+    );
+  }
+  if (columnName === "ORDER ID") {
+    return (
+      <div className= "table-cell order-id">
+        <span>{value.orderNumber}</span>
+      </div>
+    );
+  }
+  if (columnName === "ORDER DATE") {
+    return (
+      <div className= "table-cell order-date">
+        <span>{moment(value.dateActivated).format("D-MMM-YYYY")}</span>
+      </div>
+    );
+  }
+  if (columnName === "COLLECTION DATE") {
+    return (
+      <div className= "table-cell collection-date">
+        <span>{moment(value.dateActivated).format("D-MMM-YYYY")}</span>
+      </div>
+    );
+  }
+  if (columnName === "URGENCY") {
+    const urgencyClassName = cn({
+      'table-cell': true,
+      'urgency': true,
+      'stat': value.urgency === 'STAT',
+      'routine': value.urgency === 'ROUTINE',
+    });
+    return (
+      <div className={urgencyClassName}>
+        <span>{value.urgency}</span>
+      </div>
+    );
+  }
+  if (columnName === "TEST TYPE") {
+    return (
+      <div className= "table-cell test-type">
+        <span>{value.display}</span>
```

Comment:
We may want to show a different field here than just the "display", but I'm not sure, so this is fine for now!